ui/src/components/RuleCard/index.jsx

```diff
@@ -152,6 +152,7 @@ function RuleCard({
   rule,
```

Comment:
The changes to this file look unrelated to the issue you're fixing. Are they supposed to be in this PR? (If not, please revert them.)

Reply:
Changes to this file are related to previous PRs ( [PR 1](https://github.com/mozilla-releng/balrog/pull/2999), [PR 2](https://github.com/mozilla-releng/balrog/pull/3002), ) I made, approved but not merged !

@bhearsum

Reply:
Please revert them from this branch, in that case. We cannot merge this PR with unrelated changes included.

Reply:
I understand! Let me do it!

Reply:
@bhearsum, I have just reverted unrelated changes!